OOP/OOP6_ResizableArray/src/main/java/Start.java

```diff
@@ -0,0 +1,25 @@
+public class Start {
```

Comment:
Such class in which we didn't write actual implementation, but we created it to host main method was only created to perform some tests.
Since you have some tests in this PR, you don't need this dumb class any more. 
Please, don't write your code in such way from this time on.